src/components/pages/group/form/speciesCheckBox/customCheckbox/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Divider, FormControl, FormErrorMessage, FormHelperText, FormLabel } from "@chakra-ui/core";
```

Comment:
Modify Existing one to support multiple selection? or just convert it to dropdown multi select component